### PR TITLE
Add fc-dequantize squash in cpu_quantize_squash_pass for ernie model

### PR DESCRIPTION
**Pull request by lidanqing-vv**
Add fc-dequantize squash in cpu_quantize_squash_pass.

### CODE REVIEW

File: paddle/fluid/framework/ir/graph_pattern_detector.h [hunk 1]
```diff
@@ -829,6 +829,37 @@ struct ConvDequant : public PatternBase {
...
+  PATTERN_DECL_NODE(reshape_op);
+  PATTERN_DECL_NODE(reshape_out);
+
+  PATTERN_DECL_NODE(transpose_op);
+  PATTERN_DECL_NODE(transpose_out);
+
+  PATTERN_DECL_NODE(scale_op);
+  PATTERN_DECL_NODE(scale_out);
+};
+
```
Comment (wojtuss) [inline paddle/fluid/framework/ir/graph_pattern_detector.h]: Please, remove this part.

File: paddle/fluid/framework/ir/mkldnn/cpu_quantize_squash_pass.cc [hunk 2]
```diff
@@ -195,6 +195,40 @@ void CPUQuantizeSquashPass::ConvDequantSquash(Graph* graph) const {
...
+    GET_IR_NODE_FROM_SUBGRAPH(dequant_out, dequant_out, fc_dequant_pattern);
+
+    // if fc has force_fp32_output attribute
+    if (fc_out->outputs.size() == 1) {
+      fc_op->Op()->SetAttr("force_fp32_output", true);
+      fc_op->Op()->SetOutput("Out",
+                             std::vector<std::string>({dequant_out->Name()}));
+      IR_NODE_LINK_TO(fc_op, dequant_out);
+      fc_op->Op()->SetOutput("Out",
+                             std::vector<std::string>({dequant_out->Name()}));
```
Comment (wojtuss) [inline paddle/fluid/framework/ir/mkldnn/cpu_quantize_squash_pass.cc]: This is duplicated.

File: paddle/fluid/framework/ir/mkldnn/cpu_quantize_squash_pass_tester.cc [hunk 3]
```diff
@@ -374,6 +395,22 @@ TEST(CpuQuantizeSquashPass, conv_dequant_more_than_one_op_after_conv) {
...
+// a->fc->c->Concat->d
+TEST(CpuQuantizeSquashPass, fc_dequant_only_one_output) {
+  auto scale_out = 1.0f;
+  auto scale = 1.2345f;
+  auto use_mkldnn = true;
+  // remove 2 nodes: b, Dequant1
+  auto remove_nodes = 2;
+  CountNodeTest(BuildFcDequantConcatProgramDesc(use_mkldnn, scale_out, scale),
+                remove_nodes);
+}
```
Comment (wojtuss) [inline paddle/fluid/framework/ir/mkldnn/cpu_quantize_squash_pass_tester.cc:455]: Please, add verification that the `force_fp32_output` attribute is set to `true` in `fc`.

**Comment by bingyanghuang** [comment]
@lidanqing-intel Please add `test=develop` in your commit.

**Comment by lidanqing-intel** [comment]
> @lidanqing-intel Please add `test=develop` in your commit.
Hi ! @luotao1 There are some PADDLE_ENFORCE not very clear to rewrite. Should I leave it unchanged or change with my own understandings.

**Review by wojtuss — APPROVED** [review]
LGTM

**Pull request merged**